.k8s-helpers

```diff
@@ -174,12 +174,12 @@ kpfp() {
   local -r pod="${1:-$(kubectl get pod --no-headers | choose 0 | fzf)}"
   local -r data="$(kubectl get pod "${pod}" -ojson)"
   if [[ "$(jq '.spec.containers | length' <<< "${data}")" = 1 ]]; then
-    container="$(jq ".spec.containers[0].name" -r <<< "${data}")"
+    local -r container="$(jq ".spec.containers[0].name" -r <<< "${data}")"
   else
-    container="$(jq ".spec.containers[].name" -r <<< "${data}" | fzf --header='mutliple containers; please pick one')" # user choice
+    local -r container="$(jq ".spec.containers[].name" -r <<< "${data}" | fzf --header='mutliple containers; please pick one')" # user choice
```

Comment:
this is necessary too, coz in the lines below we are referring `$container` and it comes as empty without using `local -r`